front-end/src/components/UserInfoToolbar/UserInfoToolbar.jsx

```diff
@@ -34,10 +35,11 @@ export default function UserInfoToolbar() {
     dispatch({type: 'SET_MODAL', payload: false});
     clearRecipe();
   };
-
+  // app, recipe page, user
   if (user) {
-    const { name, picture, email, nickname } = user;
-    const userLink = `/user/${name}`;
+    const { name, picture, email, _id } = user;
+    // change to userID
+    const userLink = `/user/${_id}`;
```

Comment:
is userLink used? XD

Reply:
I think it's hidden by default in the review bc it wasn't part of the changes :P it gets used on line 46 + 49!